Adamant/Modules/Settings/Notifications/NotificationsViewModel.swift

```diff
@@ -192,4 +200,24 @@ private extension NotificationsViewModel {
             handler: nil
         )
     }
+    
+    func presentNotificationsDeniedError() {
+        dialogService.showAlert(
+            title: nil,
+            message: NotificationStrings.notificationsDisabled,
+            style: .alert,
+            actions: [
+                makeAction(
+                    title: .adamant.alert.settings,
+                    action: { _ in
+                        self.openAppSettings()
```

Comment:
Don't use strong references on `self` in escapings, it's easy to make a mistake with them